mobilizon_reshare/config/publishers.py

```diff
@@ -16,7 +16,13 @@
     Validator("publisher.zulip.bot_token", must_exist=True),
     Validator("publisher.zulip.bot_email", must_exist=True),
 ]
-mastodon_validators = []
+mastodon_validators = [
+    Validator("publisher.mastodon.instance", must_exist=True),
+    Validator("publisher.mastodon.token", must_exist=True),
+    Validator("publisher.mastodon.msg_template_path", must_exist=True, default=None),
+    Validator("publisher.mastodon.recap_template_path", must_exist=True, default=None),
```

Comment:
you're missing the recap header template. I just pushed in my PR a new commit. No idea how it was working before without the entry